README.md

```diff
@@ -34,19 +34,26 @@ telegraf:
           listen: ":9273"
 ```
 
-### 2. Install the Helm Chart
+### 2. Install the CRDs
 
-Install the Helm chart using the following command. This assumes you are installing it in the same namespace as your CircleCI server:
+Before installing the Prometheus operator, you must first install the Custom Resource Definitions (CRDs) required by it. Follow these steps:
 
 ```bash
-$ helm install circleci-server-monitoring-stack . --dependency-update --wait -n <your-server-namespace>
+$ helm dependency update
+$ helm install prometheus-operator-crds ./charts/prometheus-operator-crds-*.tgz
```

Comment:
>Updated the installation guide to have the prometheus-operator CRDs be installed first. The subchart has the CRDs under a templates directory instead of a crds directory. This means Helm will template the CRDs at the same time as the resources requiring them, which will fail. We could work around this with Helm hooks, but that removes them from the release and prevents Helm from uninstalling them.